tests/providers/papermill/operators/test_papermill.py

```diff
@@ -75,3 +79,4 @@ def test_render_template(self):
         assert '/tmp/out-test_render_template.ipynb' == getattr(operator, 'output_nb')
         assert {"msgs": "dag id is test_render_template!"} == getattr(operator, 'parameters')
         assert "python3" == getattr(operator, 'kernel_name')
+        assert "python" == getattr(operator, 'language_name')
```

Comment:
Could you change these asserts to use simply attribute access? e.g. `assert "python" == operator.language_name`. These `getattr` calls are non-sensical.